### DISCUSSION TITLE
Review

### DISCUSSION THREAD

**Discussion by hadoan149** [lab-brussels-1/loruki-group-5#2] [Review]
Hey group! Check this thread for the latest reviews from our coaches 😊

**Comment by hadoan149** [comment]
@EliasMlopez99 hey it seems that the images don't work again nor the icons unfortunately. Here is what @danielhalasz  said, hope it helps:

> I would recommend watching this video on solving merge conflicts:
> https://www.youtube.com/watch?v=xNVM5UxlFSA
> 
> if the images do not show up in live preview, there is probably an error in the path specified in the HTML.

**Comment by danielhalasz** [comment]
I have checked your master branch and the images and icons do work!

**Comment by danielhalasz** [comment]
![image](https://user-images.githubusercontent.com/78535490/161728913-ba591658-fd7a-4c4c-89ab-4d9f7c78837b.png)

**Reply by EliasMlopez99** [reply]
@danielhalasz Have you checked if the features html images and icons do work? Thanks beforehand

**Comment by danielhalasz** [comment]
if you change it like this, it will work:

```
<img src="./assets/hamster.gif" alt="hamster" />
